tool.go

```diff
@@ -298,11 +298,13 @@ func main() {
 	run := cmdTest.Flags().String("run", "", "Run only those tests and examples matching the regular expression.")
 	short := cmdTest.Flags().Bool("short", false, "Tell long-running tests to shorten their run time.")
 	verbose := cmdTest.Flags().BoolP("verbose", "v", false, "Log all tests as they are run. Also print all text from Log and Logf calls even if the test succeeds.")
+	buildVerbose := cmdTest.Flags().BoolP("bv", "", false, "Use a verbose build context.")
```

Comment:
Not essential to have this... but helps to verify when we get cache misses or not in running tests.

Reply:
I was wondering why we need this besides the existing `verbose`.

Reply:
ping